src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -340,14 +341,24 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         Close();
     }
 
+    // Method Description:
+    // - Creates an automation peer for the Terminal Control, enabling accessibility on our control.
+    // Arguments:
+    // - None
+    // Return Value:
+    // - The automation peer for our control
     Windows::UI::Xaml::Automation::Peers::AutomationPeer TermControl::OnCreateAutomationPeer()
     {
         Windows::UI::Xaml::Automation::Peers::AutomationPeer autoPeer{ nullptr };
         try
         {
             // create a custom automation peer with this code pattern:
             // (https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/custom-automation-peers)
-            autoPeer = winrt::make<winrt::Microsoft::Terminal::TerminalControl::implementation::TermControlAutomationPeer>(*this);
+            auto tcapInternal = winrt::make_self<winrt::Microsoft::Terminal::TerminalControl::implementation::TermControlAutomationPeer>(*this);
+            autoPeer = *tcapInternal;
+
+            _uiaEngine = std::make_unique<::Microsoft::Console::Render::UiaEngine>(tcapInternal.get());
+            _renderer->AddRenderEngine(_uiaEngine.get());
         }
         CATCH_LOG();
         return autoPeer;

```

Comment:
Could this just be `return *tcapInternal`? And if so, can we just rename `tcapInternal` to `autoPeer` and remove the other `autoPeer` local?